content/articles/how-to-embed-compose-desktop-application-to-swing-based-application/index.md

```diff
@@ -0,0 +1,236 @@
+        Row {
+            Surface(
+                color = Color(200, 200, 200),
+                modifier = Modifier.size(130.dp, 130.dp)
+            ){
+                Column {
+                    Box(
+                        modifier = Modifier.height(30.dp).fillMaxWidth(),
+                        contentAlignment = Alignment.Center
+                    ){
+                        Text("Compose Button")
+                    }
+                    Spacer(modifier = Modifier.height(50.dp))
+                    Box(
+                        contentAlignment = Alignment.Center,
+                        modifier = Modifier.height(30.dp).fillMaxWidth()
+                    ){
+                        Button(onClick = {}, shape = RoundedCornerShape(10.dp)){
+                            Text("Button 1")
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+In the `main()` function, we created Swing components of type `JFrame` and `JButton` which we added in a `ComposePanel`. `JFrame` will render the window and `ComposePanel` will act as the container to hold all the components used. 
+
+From above, we can see how the two can work perfectly with each other.
+
+Upon running this, the corresponding output will be:
+
+![output](/how-to-embed-compose-desktop-application-to-swing-based-application/compose-swing-output.png)
+
+It is worth noting the difference in the appearance of the Compose and Swing buttons. The Compose button is more appealing and better than the Swing button.
+
+### Using a SwingPanel for CFD Composition
+You can also use `SwingPanel` to hold Swing components inside a Compose application. Without further ado, let's see how this happens.
+```kotlin
+fun main() = singleWindowApplication(title = "Compose Simple Application") {
+    Column {
+        Box(modifier = Modifier.fillMaxWidth().height(60.dp).padding(top = 20.dp),
+            contentAlignment = Alignment.TopCenter){
+                Button( modifier = Modifier.size(200.dp, 40.dp), onClick = {}){
+                    Text("Compose Button")
+                }
+        }
+        Box(modifier = Modifier.fillMaxWidth().height(60.dp).padding(top = 20.dp),
+             contentAlignment = Alignment.TopCenter ){
+            Button( modifier = Modifier.size(200.dp, 40.dp), onClick = {}){
+                Text("Compose Button")
+            }
+        }
+        Box(modifier = Modifier.fillMaxWidth().height(60.dp).padding(20.dp),
+            contentAlignment = Alignment.Center){
+            SwingPanel(
+                modifier = Modifier.size(200.dp, 40.dp),
+                background = White,
+                factory = {
+                    val button = JButton("Swing Button")
+                    button.preferredSize = Dimension(40,40)
+                    val panel = JPanel()
+                    panel.add(button, BoxLayout.X_AXIS)
+                }
+            )
+        }
+    }
+}
```

Comment:
I would also consider adding comments here for beginners to better understand it